ably/export_test.go

```diff
@@ -140,6 +140,7 @@ func (c *Connection) PendingItems() int {
 	c.mtx.Lock()
 	defer c.mtx.Unlock()
 	return c.pending.Len()
+	return c.pending.Len()
```

Comment:
Looks like this was an accidentally duplicated.

Reply:
Fixed !